diff -Nur App-SVN-Bisect-1.0.ori/lib/App/SVN/Bisect.pm App-SVN-Bisect-1.0.mod/lib/App/SVN/Bisect.pm
--- App-SVN-Bisect-1.0.ori/lib/App/SVN/Bisect.pm	2009-12-04 16:57:40.000000000 +0100
+++ App-SVN-Bisect-1.0.mod/lib/App/SVN/Bisect.pm	2010-05-16 14:09:57.994672721 +0200
@@ -476,6 +476,8 @@
 sub cmd {
     my ($self, $cmd) = @_;
     $self->verbose("Running: $cmd\n");
+    local $ENV;
+    $ENV{LC_ALL}='C';
     my $output = qx($cmd);
     my $rv = $? >> 8;
     if($rv) {
